hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-router/src/test/java/org/apache/hadoop/yarn/server/router/clientrm/TestableFederationClientInterceptor.java

```diff
@@ -71,7 +75,8 @@ protected ApplicationClientProtocol getClientRMProxyForSubCluster(
         mockRM.init(super.getConf());
         mockRM.start();
         try {
-          mockRM.registerNode("h1:1234", 1024);
+          MockNM nm = mockRM.registerNode("127.0.0.1:1234", 8092,4);
```

Comment:
8*1024, 4

Reply:
Thanks for your help reviewing the code, I will fix it.